src/lib/crypto/kimchi_backend/gadgets/foreign_field.ml

```diff
@@ -816,7 +818,7 @@ let%test_unit "foreign_field_mul gadget" =
                   (module Runner.Impl)
                   product
               in
-              assert (product = expected) ) ;
+              assert (Stdlib.(product = expected)) ) ;
```

Comment:
Wondering whether the code below is not less error-prone - it probably checks the same thing but you never know ;-)
```suggestion
               let assert_eq ((a, b, c) : field standard_limbs)
                  ((x, y, z) : field standard_limbs) =
                let open Runner.Impl.Field in
                Assert.equal (constant a) (constant x) ;
                Assert.equal (constant b) (constant y) ;
                Assert.equal (constant c) (constant z)
              in
              assert_eq product expected  );
```

Reply:
I prefer safety.  Thanks!